pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -99,7 +102,7 @@ func (c *Cluster) configureDashboardModules() error {
 		}
 	}
 	if hasChanged {
-		logger.Infof("dashboard config has changed. restarting the dashboard module.")
+		logger.Info("dashboard config has changed. restarting the dashboard module.")
```

Comment:
Nit. Per golang syntax guidelines.

```suggestion
		logger.Info("dashboard config has changed. restarting the dashboard module")
```